From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tiberiu Chibici <chibicitiberiu@gmail.com>
Date: Fri, 30 Oct 2026 01:18:31 +0200
Subject: Add loom-ega-es-data, the Spanish EGA floppy release of Loom

The Spanish EGA release of Loom was not known to gdp. ScummVM knows it
(engines/scumm/scumm-md5.h: 00.LFL 2a208ffbcd0e83e86f4356e6f64aa6e1, loom
EGA ES_ESP DOS). It has the same 74 room files as the English, German and
French EGA releases; 28 of them are byte for byte theirs and keep their
names here, the other 46 (the ones with text) are new, with sizes and
checksums from an owner's copy. The English 1.1 patch (EGA patch) is not
offered for it.

Forwarded: not yet (candidate for Debian)

diff --git a/data/loom.yaml b/data/loom.yaml
--- a/data/loom.yaml
+++ b/data/loom.yaml
@@ -61,6 +61,13 @@ packages:
     optional:
     - EGA patch
 
+  loom-ega-es-data:
+    provides: loom-data
+    lang: es
+    longname: "Loom (EGA)"
+    install:
+    - EGA assets spanish
+
 files:
   000.lfl_provider:
     alternatives:
@@ -388,6 +395,82 @@ groups:
     4203      4370a15190d7261ddcffd894f0f78c85 87.lfl
     2168      96be1d62f87082054f6459cf4916d85b 99.lfl?fr
 
+  EGA assets spanish: |
+    5748      2a208ffbcd0e83e86f4356e6f64aa6e1 00.lfl?es
+    55394     0d3fb604a234e48861779ddc800b5f99 01.lfl?es
+    58815     a73656bfc5dc998860f584c702c25d94 02.lfl?es
+    14606     3e8a478d922d2d1d3b8060f2cf5bfcb2 03.lfl?es
+    33669     37c01f3e399ed9e860ecd4b4bcc4f564 04.lfl?es
+    26175     6cc5b8b6c1fed8a4cfca09117afd989f 05.lfl
+    25632     847ec789f59f1d729a6da9b800606137 06.lfl?es
+    48146     9e153370e86c5a4dac13e07becba69ff 07.lfl
+    55382     df0e6dac544515ebbfc7749eef317f0f 08.lfl?es
+    57050     ced20ba485f58a46cf2db1d615165aae 09.lfl?es
+    34166     d2df047b5536707b0057354f06304e74 10.lfl?es
+    29513     f7560fd49cc094f2bcc5460126fe474f 11.lfl?es
+    20394     dfa54a399548c956e7f3c29c1916109b 12.lfl?es
+    45740     f77c4de0af80a7f9819cb123ff5f51c6 13.lfl?es
+    21573     b43d019579658a78c7d238a590254ff9 14.lfl?defr
+    22071     6cba6a1da3cbb6cd58c9a4e96a28ca83 15.lfl?es
+    47870     d5517e726869fd69f343de1d4ae1726f 16.lfl?es
+    28358     bc4706f0e9825fef4d4052f6cee6a42e 17.lfl?es
+    18903     3f8ed2dfa7521a2b1505f21429ef902f 18.lfl?es
+    33770     d5eba4da956c39d7511a42884ddc23ee 19.lfl?de
+    37634     6c2d3df2711a80075b86728bddc2bf5c 20.lfl?es
+    21485     9f65f510db6bd0576931edc29d92bf5d 22.lfl?es
+    37456     b279ddd2565323cbcf186f367bf3ffed 23.lfl?es
+    52964     fae3304b2aa0bdebac01586073c82eec 24.lfl?es
+    47350     553742e33a185837b8dfc73922a5513b 25.lfl?es
+    25618     4d55d34fdd380e8ecb898dfed498e9be 26.lfl?es
+    13295     9be46ff2fc74634b5c8add406ac9a08a 27.lfl
+    45466     46febf5f22ff63a57983d5f38b0261dc 28.lfl?es
+    5752      f3970e996fa121dc0c25c2c0aab07e28 29.lfl
+    26619     23255cb90d599fcb4fba01b0faeb8902 30.lfl?es
+    29291     f918dd2c1fcaa763613aa8205cf702bd 31.lfl?es
+    7580      a7cfb94833c2b4341849e7a1db983dcf 32.lfl?defr
+    52793     7f0356d8244ab98ad7837a68400fcc68 33.lfl?es
+    59031     bf18cfcb766633520c3737fbbc2e19fe 34.lfl?es
+    36525     b9aea60982d00ed085115fcf0f0a9145 35.lfl?es
+    41462     e8bbaba228370e1f938dd291f94c46d8 36.lfl?defr
+    29346     767a530537027c6ac48c7ed494d6bf25 37.lfl?es
+    15134     858f898f2d4f603f0f271dd372654883 38.lfl?es
+    5717      c85151c977755be8515793af2142b947 39.lfl
+    15383     c60e08b956a38490246d793ab99fb211 40.lfl?defr
+    36279     090cc898b7e1c72d63ff8d8d081a00a8 41.lfl?es
+    39042     e7dd32e3eef7e9fcf119100a4fd92b3f 42.lfl?defr
+    49677     5b0b574da431e1f3ae443541826af8c5 43.lfl?es
+    32715     17c927c01f5593c5d1226516594eded6 44.lfl?es
+    46527     43831036371ae80a74302f13fc8cfe45 45.lfl?es
+    54123     64667985c9a6054b8f87683369d3e821 46.lfl?es
+    33942     ebce07a1dbb3f5b2ff15a683758262fd 47.lfl?es
+    27566     242167ae0123e1215eaf68685830b3ee 49.lfl?es
+    18047     0d31788f72b7cd34797ff4ec5523f3f7 50.lfl
+    22925     f8a719db176412c7d95f1a36f098ee6b 51.lfl?es
+    6924      a5f2416b7ba2368bef7a0831cc752d48 52.lfl
+    44067     50614b29f27f6632a3e110230682a513 53.lfl
+    4670      5888dcbb815a84170548162fdfe8fd23 54.lfl
+    4636      ccd012cde52ece057105011f249eb1c0 55.lfl
+    7555      6d50e5cdfe1d4be52653a1e208578b17 56.lfl?es
+    5450      c6e28794db5c2ef16b215ee0a7ec5a57 57.lfl
+    9305      f11b534ae85dff80b3ad3a4e38509ce2 58.lfl
+    40760     88e1889d9e6750c8285dace3ad0e684b 59.lfl
+    4169      b0c30e95f51f1f1298fc075e4ae1dd2e 60.lfl
+    10898     5d20afa764680610125f6519680ecbb4 61.lfl
+    16443     c12f7136dbc6d61eb70fd088e04476c8 62.lfl?es
+    3852      12c85db087f354a22d3fdac45d9cada3 63.lfl?es
+    9241      1002c2b2ea982c29cc4ab1755b542f89 64.lfl?es
+    1778      b21cc11869da654872d787603b329eab 65.lfl?es
+    32498     058a2812f676ab18bbe8a1b0dcfda52b 66.lfl?es
+    4386      0771b462de487778e4d7cb1ba166abae 67.lfl
+    10891     65d0b7a23b5c92ec22092170168b9e31 68.lfl
+    52618     13c0663e268cef972797af09455a3082 69.lfl?es
+    11259     f6f3a87936cf5b3a80d657fd779750b9 70.lfl?es
+    6515      e6dddf862f076402cb0aa6aa28b35405 79.lfl
+    11889     c517db260ad1a6ee41587fa41b1e4485 81.lfl
+    22466     dfd77f697ed8e79b138d993c33069ed7 86.lfl
+    4203      4370a15190d7261ddcffd894f0f78c85 87.lfl
+    2168      96be1d62f87082054f6459cf4916d85b 99.lfl?fr
+
 sha1sums: |
   366ce8bb2c7876a445c870f6b19f8d56cbf91f47 000.lfl
   e7b80c90023b85d77349861d7a4f8c68b7b3b6d7 901.lfl
@@ -586,6 +669,53 @@ sha1sums: |
   a869eefe0a50d64ad9a825977c90ea5304b261e0 loomup.zip
   562ef8509f1cbf4566516e0955c0eacc037e0951 loomup.exe
 
+  48a4ecd909132062443780bf7d38e1c0a2358342 00.lfl?es
+  82ce4f1be99c2853af56bbbcef783f8b8427be3b 01.lfl?es
+  da441452cd36f829edd7b1fe25017bacf60651ac 02.lfl?es
+  7143c2319db3bcb6e683053b93d8f3a5d51aa59e 03.lfl?es
+  dcc1dfd430274bed6bceb9640c6d01d337a1178d 04.lfl?es
+  c3bb4efedfa29bd3a78edc8142aab063eb668fa4 06.lfl?es
+  1fe93f1ea8e29d8d8fd38db485dc0ac716274f6f 08.lfl?es
+  db057e81586762911d22f122c016aba4c4bfd96a 09.lfl?es
+  d5a472d60b38f4ea14eac52a574512d62c008f9e 10.lfl?es
+  2cbb9227110b21f5279541d5866005eeb53dd168 11.lfl?es
+  cbff6152704c498e7973ffd6b4dc097c69bb1da4 12.lfl?es
+  0b414bc02c3eed9ac47cbe558617d2527bc2e8e3 13.lfl?es
+  1ad39114058666899385bb5933c4440031b36ceb 15.lfl?es
+  4294e900a0a40f7e19c84b62fc71f43cd1c194be 16.lfl?es
+  9457e8389998d7890d00b62861e9b06c157616d4 17.lfl?es
+  912df1371d844ad333a676974b1a73a2e1acbfec 18.lfl?es
+  9847754eef36d46e5b5fb04da43326b925404a0e 20.lfl?es
+  ba062638b5e3013c957dbf3e9e6a158b426a3f5f 22.lfl?es
+  3e462a1ace3b5ba5b06ff15dcad0918baaa25876 23.lfl?es
+  a62091d407448884f64a4f228e730ab1e24052b2 24.lfl?es
+  c2fd0f1450bc167b18652c8e7e60e6a0a6702e52 25.lfl?es
+  788d996fa3a1f3d1f0a30aaa6cc8f52d67e14843 26.lfl?es
+  02a9533320efd40d7bf8a36f5cf771b1fc36f0b7 28.lfl?es
+  02d474bc718793bee06071d7fcb5a18460437746 30.lfl?es
+  3039f3a0b004c534bf22d3881e1ed989ded0ae49 31.lfl?es
+  2bcc50c5099aed2a449246fb2c445b5465021035 33.lfl?es
+  df7eba5dc8855e96dd7f803722fa25d23b6a2148 34.lfl?es
+  fe8dfd8d2b8677d0502e5c8840b13b95cc399e0d 35.lfl?es
+  8733bed0c1757ae6b93633ad322aa661f847f2fa 37.lfl?es
+  472cdb699a3d2d56da02db3b1523145fd6d2f56a 38.lfl?es
+  b800c0dd9e4e0df4bf1141f83d6b9e2d5bae4785 41.lfl?es
+  f1621fd7642bcca0f2b010e23eae31cc4aed7989 43.lfl?es
+  165a0b7379129da2d98b14183cbaf197b1720444 44.lfl?es
+  a96a4030bbd5c6f172d8c6f7a61f53e1015f1eec 45.lfl?es
+  a07077e1d030fd65b80927f29af8091ddcc71e6d 46.lfl?es
+  d45694bfa030db3c58963197bdcc3c930b2a2907 47.lfl?es
+  072fe7c8de354af503faca812dc5b009f0c56d63 49.lfl?es
+  2b8f8f8818550124039b021efea50d78831b0ba6 51.lfl?es
+  c1f1165bc949a91767489255fb6cd4c2a4e003ec 56.lfl?es
+  adbe3b374a17096c00bb08b7d9f3b5572229255d 62.lfl?es
+  444c3651d7ae1405ec12bd9ea8bc7767f4df04ab 63.lfl?es
+  a8df63cfd96d6605a3b03a419dd928495d748bfa 64.lfl?es
+  1a80244da245745eb25494930e4756cc1b2646af 65.lfl?es
+  49bd39d4f06a25d04e3ee5e397e5d3b23237f6e5 66.lfl?es
+  1f754300205e83cc20c43d22c00a3c2007863073 69.lfl?es
+  677762e0143f966d96ba83d7bbc9966f65bbf752 70.lfl?es
+
 sha256sums: |
   a13d9b5d5ebda838014293e5a9bf131d4bf7dc792dc3a0ae193838b8b3395a05 000.lfl
   ca577a8ad47980ab83319504c26261b5ac4fcdf7f2eeceb1a0ed437f9b5c4643 901.lfl
@@ -723,4 +853,52 @@ sha256sums: |
   d3ce48d7395db63a222e02a378370bcebe5b786218efaee0ceb0e7aa7b339191 87.lfl
   f89738be02bb1c711f7465453251fbce07a81cc6b8150128c079383acd4b9d71 99.lfl?de
   9a8d26a5d96e111ad10e9ffa0a4d2ee050a845b905c137127c3ffca9607409f7 99.lfl?fr
+
+  e08bea3d558a96f8d5534d3b499a393db6166e15c27e3564a2dc8d47c4213ee9 00.lfl?es
+  e7dacd95a2ac82df07f28d7277ace9882ec69c49009452da8253b80d41ff9721 01.lfl?es
+  34dfca248d0a5870297d52aaf777838e3a00f680b618c6a299355f015f742068 02.lfl?es
+  5d95ec54d99bf54d7c244eeb572ad452d3c005f68245482ac3959833c8dfcb9a 03.lfl?es
+  747d02ba0b57f2bd3a93a59fe1a9f98d753e675131f27323e105e867f1ad0426 04.lfl?es
+  5361464e43850c200674614308ea5fa6556b976c83f0b4967c5e475fcb92ac35 06.lfl?es
+  e74d9f904374760287ac13d6b97a056ce344fa1fdd79543c0b0bb2c3ea12c3ee 08.lfl?es
+  d8e638cea86ddee87a839ceab9bfef5f9c11f4361020bc21bc79f4f5f7fadad4 09.lfl?es
+  57f9246dbd7cd3e74b52f7948468c90abd8422a133703e33b952e93e4a9426a0 10.lfl?es
+  4ea94cd412df3061c413a55ae4d84a9fa09b89f493745d755e8b8ee69acfca50 11.lfl?es
+  2d1547da39e6764e1e9aa6df36b7a0b438056c906bec08a48c4d92e00a5d4739 12.lfl?es
+  af8d33301766bc819b3b75d52f53ba6882bbc7d6639699cb25bd764db723fb27 13.lfl?es
+  be2931cda6e2058758b98bf41d1f08f1dd7468f79c1de545e50aef60c31c8353 15.lfl?es
+  7e2b9ba5bd213448afbc6f2ccb702a0dbbcf60c018d91ff9a1e942f7859eca28 16.lfl?es
+  55ff1c4b25cde8f96cfe95addc7d6095e6627e45bd8e6d5affa71617d2800908 17.lfl?es
+  454b447a07f0d8e62d56f25eb14f7a69b77ca78b9a8f8baaf5afb50cc9e0b17e 18.lfl?es
+  9e7d2aadd362d0dd9619c34a569ec86bf8e15db85faa855cdf9691d4d8c96484 20.lfl?es
+  ca81f4c8fb9d7330c27499dd4ec44f3c936765a5f55a5a1007d6347e73c0634a 22.lfl?es
+  1d621689299c85c8dd64919f79dc9dd67b8f63850df191c2bfe5fa64e997c9f7 23.lfl?es
+  30b4d5975c7f60c6b18cb380b5db54f7aaa6000ce7306c5e1374ac923c7a0793 24.lfl?es
+  b2dcd8ba37b790f1a5e2dc14a8260b709650e97620d1f7bf792afe3a06a67cd2 25.lfl?es
+  f614666ebe4d34b76cc65900d8b58a8cc0f6eefc163ba28738075a4d9ebb17c2 26.lfl?es
+  19426a50856dcaed60eade8c0a000fe7e358350ef5d1a10e7fbed4fb3eeccac8 28.lfl?es
+  09cfe7432ac792279af31e961c76f726abcd2b1c7bc17e15c1b53fa6cb05703d 30.lfl?es
+  9dfd8f8bb8eea38fdcc2c7c411f4cce9db27aa0c6b27f1335654863329973123 31.lfl?es
+  8d2e56f8dbfb3e6e530ead13f2ebadb98fb59f6a0a1364b443401c79f484d57b 33.lfl?es
+  083a1ae30c73b49fad9d3fda2344adcbfbbfc5db20318e9332036de85aaa7b89 34.lfl?es
+  943c82a2374202682f459a211fbabde2e86316280e5d829df8e4f6f769dbb9fb 35.lfl?es
+  b930670ddf12fc987e2dcbb0e27a4be3455f6671d0c4bf7dc776a73f959d73d8 37.lfl?es
+  bf8b604867ab60a7fe21146b4f6b34a5f8188f4783e74a927cb7a3db15a267a9 38.lfl?es
+  21e69bfd42e6ba680e52ff54e6c825c4f83b3be4a5f01c2624a96695ce479188 41.lfl?es
+  87c6f1b14e01213dc5ab150ad6a7e1894795c64fb5cde8543ed240e0abec6fb4 43.lfl?es
+  1da8cf8e2d129fae42c6aac624fdeac627049c62b3bdff390109807e13d49734 44.lfl?es
+  79c22bdca86e925816d62745a679aec2464fd23d4b84192eee80a4a357fb4302 45.lfl?es
+  2f07dba3ff41e0389329a43de63fc02e9a26fa9de7f7fe7ae4fea060783e3342 46.lfl?es
+  087696ba12599c24a37bf79467f572f757da3183be1b0f5b1aa2a5bcf09d2e1d 47.lfl?es
+  11c8aaef21431c7a95ea24c82249e9838b7cc531349f3a34921104ad6f2fd4fe 49.lfl?es
+  44efd2af97b88acb900f5a1edb0cbb27807d86efe7b63abc325e2c0dd62781bd 51.lfl?es
+  ee36a0451cb72f7f1e7781534336dfea8da13b34b77a3d5ff7ad965bbbdc18c9 56.lfl?es
+  a32cf6898b45c531140490eb06ad547034b5a8b24eb8fc74997e6e56a687fc4a 62.lfl?es
+  c011ec65cd8d694794ebbc298981499bbfac912e5bdfd71be1ee90006e64c6bf 63.lfl?es
+  fdf9dd7ab6625237de766ae2fae81c3b466a0330e40d5ee384d110db9b104339 64.lfl?es
+  bb385c41711de54fd7207e92d627640fadba5249d9f978583ee8d222538724cf 65.lfl?es
+  4a947aac812ca8192494d70a764dbf0334751391c8eef6f577f5de92d13108df 66.lfl?es
+  4b430d2cd56f29e2502ad622d0295495904d788fa338338d928bc1d9ba2a991c 69.lfl?es
+  1323cc4cc8c0be2677a339f0c41eba7f24f7cb345b8355c1124e9d316373b948 70.lfl?es
+
 ...
